fourm/data/modality_transforms.py

```diff
@@ -706,13 +748,51 @@ def image_augment(
                 "Crop settings / augmentation index are missing but a pre-tokenized modality is being used"
             )
         v = torch.tensor(v[rand_aug_idx])
-        return v # Since we augment before saving (create 5 augmented versions of the same image), in this transform you randomly sample one of the augmentations to work with. 
+        return v  # Since we augment before saving (create 5 augmented versions of the same image), in this transform you randomly sample one of the augmentations to work with.
 
     def postprocess(self, sample):
         return sample
 
 
-class DetectionTransform(AbstractTransform): # bounding boxes
+class VideoTokTransform(AbstractTransform):
+    """
+    Assume input tokens is an ndarray of shape (num_frames, num_tokens_per_frame).
+    Transform the tokens to a torch tensor of shape (num_frames * num_tokens,).
```

Comment:
Maybe don't flatten it here because this will make it a little bit more annoying to add the temporal embedding. Instead we can unflatten after adding the temporal embedding.